client/src/components/DataFiles/DataFilesModals/DataFilesPreviewModal.jsx

```diff
@@ -1,8 +1,26 @@
-import React, { useState, useCallback } from 'react';
+import React, { useState, useCallback, useRef, useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { Modal, ModalHeader, ModalBody, Button } from 'reactstrap';
 import { LoadingSpinner, SectionMessage } from '_common';
 import styles from './DataFilesPreviewModal.module.scss';
+import { Niivue } from '@niivue/niivue';
+
+const NiiVue = ({ imageUrl, fileName }) => {
+  const canvas = useRef();
+  useEffect(() => {
+    const volumeList = [
+      {
+        url: imageUrl,
+        name: fileName,
+      },
+    ];
+    const nv = new Niivue();
+    nv.attachToCanvas(canvas.current);
+    nv.loadVolumes(volumeList);
+  }, [imageUrl]);
+
+  return <canvas ref={canvas} height={480} width={640} />;
+};
```

Comment:
Should we move this out to its own component file? Just to keep separation of general vs specific functionality

Reply:
I see your point but I don't want to drop another component file into this mess: 
<img width="438" alt="image" src="https://user-images.githubusercontent.com/12601812/224826704-bada9250-4fb9-4a42-a894-82d68ea3dbc7.png">
It makes sense to me to keep these components in the same file because there's no other context where the NiiVue stuff will ever be used.

Reply:
Point taken